src/main/java/com/sonyericsson/jenkins/plugins/bfa/BuildFailureScanner.java

```diff
@@ -175,6 +175,10 @@ public static void scan(Run build, PrintStream buildLog) {
                 printDownstream(buildLog, downstreamFailureCauses);
             }
 
+            if (PluginImpl.getInstance().isEnableBuildDescription() && !foundCauseList.isEmpty()) {
```

Comment:
I find it strange why this would be a global only option.
But I guess adding a per project setting is a bit too much work since we don't have any else of those at the moment. So it can be added later as a `JobProperty` if needed.